### ISSUE TITLE
Minified languages file assumes global "exports" variable exists

### ISSUE THREAD

**Issue by mkleehammer** [foretagsplatsen/numbro#146]
The minified 1.7.0 file has the following on line 179, between the Hungarian and Italian translations:

    exports["cs-CZ"]=require("./cs-CZ.js"),exports["da-DK"]=...

It includes quite a few languages - possibly all of them.

Since I am including the file statically, there is no `exports` global variable.  While I can work around it, I'm assuming it is an error just because of the odd placement in the generated file.

**Comment by mmollick** [comment]
This was a bug I introduced when making the package compatible with webpack. The PR fixing it is waiting to be accepted.
https://github.com/foretagsplatsen/numbro/pull/143

**Comment by BenjaminVanRyseghem** [comment]
I merged @mmollick PR, so should we close this issue?

**Comment by mmollick** [comment]
I think it's going to need a patch release for npm.

**Comment by BenjaminVanRyseghem** [comment]
indeed :smile: 

I will try to do that today

**Comment by phpguru** [comment]
What's the status on this?

**Comment by BenjaminVanRyseghem** [comment]
an update is needed

**Comment by phpguru** [comment]
What's the workaround? @mkleehammer

**Comment by mmollick** [comment]
I don't believe @BenjaminVanRyseghem has gotten around to making a patched release containing #143

**Comment by BenjaminVanRyseghem** [comment]
done
